src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -1,24 +1,61 @@
 import React from 'react';
-import { Table, TableHeader, TableBody, expandable } from '@patternfly/react-table';
+import { Table, TableHeader, TableBody, expandable, TableComposable, Thead, Tr, Th, Td, Tbody } from '@patternfly/react-table';
 import propTypes from 'prop-types';
 import { flatMap } from 'lodash';
 import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
 import { EmptyStateDisplay } from '../EmptyStateDisplay/EmptyStateDisplay';
-import { CheckCircleIcon } from '@patternfly/react-icons';
+import { CheckCircleIcon, BullseyeIcon, WrenchIcon } from '@patternfly/react-icons';
 import { TextContent, Text, TextVariants } from '@patternfly/react-core';
 import './RecommendationsTable.scss';
 
 const renderExpandedView = (row) => {
     return (
         <TextContent>
-            <Text component={TextVariants.p} className="newline tab">
-                <Text><strong>Detected issues</strong></Text>
+            <Text component={TextVariants.p}>
+                <Text className="margin-text-bottom"><BullseyeIcon/><strong>Detected issues</strong></Text>
                 {row.reason}
             </Text>
-            <Text component={TextVariants.p} className="newline">
-                <Text><strong>Suggestion</strong></Text>
+            {row.detected_issues && <TableComposable
+                arial-label="Detected issues table"
+                variant="compact"
+                borders={false}
+                className="table-border-top detected-issues-table"
+            >
+                <Thead>
+                    <Tr>
+                        <Th>Identified issues by ROS</Th>
+                    </Tr>
+                </Thead>
+                <Tbody>
+                    <Tr>
+                        <Td dataLabel="Detected Issues" className="newline tab">{row.detected_issues}</Td>
```

Comment:
@r14chandra I think the dataLabel should be `Identified issues by ROS` here since `Detected issues` is mentioned above. Let me know what you think.

Reply:
On digging it further, found that dataLable is used for mobile view. So, it'll show `Detected Issues` as the column name on mobile view, which is not correct. Its value should be same as mentioned under `Th` tag. I'll fix it. Thanks!